services/movie.py

```diff
@@ -6,20 +7,24 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
 ) -> QuerySet:
     queryset = Movie.objects.all()
-
     if genres_ids:
         queryset = queryset.filter(genres__id__in=genres_ids)
 
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
```

Comment:
If you decide to modify it, it would be great if you could also update the return annotation to include None as a possible return value.